crates/shared/src/arguments.rs

```diff
@@ -141,9 +141,21 @@ pub struct Arguments {
     #[clap(long, env, default_value = "http://localhost:8545")]
     pub node_url: Url,
 
-    /// The base URL used to connect to subgraph clients.
-    #[clap(long, env, default_value = "https://api.thegraph.com/subgraphs/name/")]
-    pub graph_api_base_url: Url,
+    /// The Balancer subgraph URL
+    #[clap(
+        long,
+        env,
+        default_value = "https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2"
```

Comment:
It's not great that this default now only works for mainnet. 
I wonder if it would be better to make this value Option and implement the default based on the current connected chain similar to how we have it currently

Reply:
Since the graph is shutting down for good there is no public URL that we could be using for these defaults.
Will make all values optional and only enable the liquidity source if the URLs have been set. Also there are a bunch of old and ignored tests that need the public URLs. I will delete them since they will no longer be functional.